Transactions/Transaction.cs

```diff
@@ -11,12 +12,27 @@ public class Transaction
         public string TransactionType { get;}
         public float Amount { get; }
         public string Title { get; }
+        public string Category { get; set; }
 
-        public Transaction(string transactionType,  string title, string amountStr)
+
+        public Transaction(string transactionType, float amount, string title, string category)
         {
+            
+
             this.TransactionType = transactionType;
             this.Title = title;
-            Amount = float.Parse(amountStr);
+            this.Amount = amount;
+            this.Category = category;
+        }
+
+        public void writeAmount()
+        {
+            Console.WriteLine(this.Amount);
+        }
+
+        public void changeCategory(string category)
```

Comment:
same as above
also, there's no real need to have a separate setter method, when the setter of property Category is public (line 15). As the calling class can just use the code below to update category:
`transaction.Category = "New Category"`